commander/src/commands/transaction/create/unlock.ts

```diff
@@ -0,0 +1,189 @@
+import { flags as commonFlags } from '../../../utils/flags';
+import { getNetworkIdentifierWithInput } from '../../../utils/network_identifier';
+import { getPassphraseFromPrompt } from '../../../utils/reader';
+
+interface RawAssetUnlock {
+	readonly delegateAddress: string;
+	readonly amount: string;
+	readonly unvoteHeight: number;
+}
+
+const createUnlockTransaction = (
+	nonce: string,
+	fee: string,
+	networkIdentifier: string,
+	unlockingObjects: ReadonlyArray<RawAssetUnlock>,
+	passphrase?: string,
+) =>
+	unlockToken({
+		nonce,
+		fee,
+		networkIdentifier,
+		passphrase,
+		unlockingObjects,
+	});
+
+interface Args {
+	readonly nonce: string;
+	readonly fee: string;
+}
+
+const splitInputs = (unlock: string) =>
+	unlock
+		.replace(/\n/g, ',')
+		.split(',')
+		.filter(Boolean)
+		.map(u => u.trim());
+
+const validateUnlocks = (
+	unlocks: ReadonlyArray<string>,
+): ReadonlyArray<RawAssetUnlock> => {
+	const rawAssetUnlock = [];
+	for (const unlock of unlocks) {
+		const [delegateAddress, amount, unvoteHeight] = splitInputs(unlock);
+		if (!validateAddress(delegateAddress)) {
+			throw new ValidationError('Enter a valid address in LSK string format.');
+		}
+		if (!isNumberString(amount)) {
+			throw new ValidationError('Enter the amount in valid number format.');
+		}
+		if (!isNumberString(unvoteHeight)) {
+			throw new ValidationError(
+				'Enter the unvoteHeight in valid number format.',
+			);
+		}
+
+		rawAssetUnlock.push({
+			delegateAddress,
+			amount,
+			unvoteHeight: parseInt(unvoteHeight, 10),
+		});
+	}
+
+	return rawAssetUnlock;
+};
+
+export default class UnlockCommand extends BaseCommand {
+	static args = [
+		{
+			name: 'nonce',
+			required: true,
+			description: 'Nonce of the transaction.',
+		},
+		{
+			name: 'fee',
+			required: true,
+			description: 'Transaction fee in LSK.',
+		},
+	];
+	static description = `
+	Creates a transaction which will unlock tokens voted for delegates and add them back to the voter balance.
+	`;
+
+	static examples = [
+		'transaction:create:unlock 1 100 --unlock="123L,1000000000,500"',
+		'transaction:create:unlock 1 100 --unlock="123L,1000000000,500" --unlock="456L,1000000000,500"',
+	];
+
+	static flags = {
+		...BaseCommand.flags,
+		unlock: flagParser.string({ ...commonFlags.unlock, multiple: true }),
+		'no-signature': flagParser.boolean(commonFlags.noSignature),
+		networkIdentifier: flagParser.string(commonFlags.networkIdentifier),
+		passphrase: flagParser.string(commonFlags.passphrase),
+	};
+
+	async run(): Promise<void> {
+		const {
+			args,
+			flags: {
+				unlock: unlocks,
+				networkIdentifier: networkIdentifierSource,
+				passphrase: passphraseSource,
+				'no-signature': noSignature,
+			},
+		} = this.parse(UnlockCommand);
+
+		const { nonce, fee } = args as Args;
+
+		if (!isValidNonce(nonce)) {
+			throw new ValidationError('Enter a valid nonce in number string format.');
+		}
+
+		if (!isNumberString(fee)) {
```

Comment:
fee is in `LSK`, and when inserting the decimal number here, it fails

Reply:
`if (Number.isNaN(Number(fee))) {`